code/modules/jobs/job_types/roguetown/refugees/types/rogue.dm

```diff
@@ -108,10 +109,21 @@
 			armor = /obj/item/clothing/suit/roguetown/armor/leather
 			cloak = /obj/item/clothing/cloak/raincloak/mortus
 			backl = /obj/item/storage/backpack/rogue/satchel
-			backr = /obj/item/gun/ballistic/revolver/grenadelauncher/crossbow
 			beltr = /obj/item/rogueweapon/huntingknife/idagger
-			beltl = /obj/item/ammo_holder/quiver/bolts
 			backpack_contents = list(/obj/item/storage/belt/rogue/pouch/coins/poor = 1)
+
+			var/weapons = list("crossbow", "bow")
+			var/weaponschoice = input("Choose your weapon", "Available weapons") as anything in weapons
+
+			switch(weaponschoice)
+
+				if("bow")
+					backr = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/recurve
+					beltl = /obj/item/ammo_holder/quiver/arrows
```

Comment:
Set the default level for both crossbows and bows to 2, and add a one level skillup for each weapon for the choices separately.